README-EN.md

```diff
@@ -1,64 +1,103 @@
 To learn more about the Code for Development initiative, visit: www.code.iadb.org
 
-### IDB OMK Pilot
+### IDB OpenMapKit Pilot
 ### Description and Context
 ---
 
-OpenMapKit pilot implementation for IDB
+OpenMapKit (OMK) allows users to complete data collection surveys in the field that contain geospatial information. In addition to an Android device, there are three software components that makes this possible:
+
+- OMK Server
+- OMK Survey App
+- OpenDataKit (ODK) Collect App
 
 ### User Guide
 
-#### Server
-An OMK server has been setup using a HOT OSM Fork:
-* https://github.com/hotosm/OpenMapKitServer/
+#### Creating a survey
+
+OMK adds geospatial data collection capabilities to [ODK surveys](https://docs.opendatakit.org/) by extending the [XLSForm standard](http://xlsform.org/en/) used by ODK for specifying surveys with OSM related fields.
+
+OMK provides [documentation](http://openmapkit.org/docs_odkformsforomk.html) for creating a survey, as well as a sample survey [form](https://docs.google.com/spreadsheets/d/11H4-mGYTS61GLjSbVoTbmhoI5DjlF5fcBwNwQcvd2Go/edit#gid=0).
+
+Once a survey form has been created, it needs to be uploaded to the OMK server. Login to the OMK administration dashboard, and click the "Upload Form" button in the header.
+
+<img width="371" alt="Screen Shot 2019-03-19 at 11 49 39 AM" src="https://user-images.githubusercontent.com/1042475/54620928-4efa5580-4a3d-11e9-9f20-65a2c3418a95.png">
+
+If the form does not contain any errors, you should see the following message.
+
+<img width="1158" alt="Screen Shot 2019-03-19 at 11 54 00 AM" src="https://user-images.githubusercontent.com/1042475/54621169-b57f7380-4a3d-11e9-8ad9-ab277e46ab8b.png">
+
+#### Filling out a survey
 
-Additional changes had to be made to get it operating successfully on our EC2 instance
-* https://github.com/hotosm/OpenMapKitServer/compare/master...rbreslow:feature/jrb/add-compose-project
+To fill out the survey you created, you must first point the ODK Collect App at the IDB OMK server. This can be set under the Server section of the General Settings menu.
 
-Administrators can access the site at
-* http://idb-omk.azavea.com/omk/pages/#/
+<img src="https://user-images.githubusercontent.com/1014341/50118987-639d3680-021f-11e9-910d-a0509d9b4f76.png" width="300px">
+
+Back on the app home screen, click the "Get Blank Form" button. This will download a list of all the forms available on the server. Once the list is populated, select the form you want and click "Get Selected". The form should now be downloaded to your device.
+
+From the home screen, click "Fill Blank Form", and select the survey you would like to complete. The survey should now launch. Swipe left to proceed through the survey questions.
+
+When filling out the survey, any geospatial questions will launch the OMK app, where you'll be able to add nodes with the attributes you specified in the survey form.
```

Comment:
Can we mention here:
- "nodes" means only points in this context, can't add lines/polygons
- differentiate that attributes on the node features are different than survey questions. Or rather, they can both be specified in the survey, but some are questions and some are attributes on the OSM feature (at least that's my understanding)